tools/Azure.Mcp.Tools.KeyVault/src/KeyVaultSetup.cs

```diff
@@ -34,6 +35,8 @@ public void RegisterCommands(CommandGroup rootGroup, ILoggerFactory loggerFactor
 
         var certificate = new CommandGroup("certificate", "Key Vault certificate operations - Commands for managing and accessing certificates in Azure Key Vault.");
         keyVault.AddSubGroup(certificate);
+        var admin = new CommandGroup("admin", "Key Vault administrative operations - Management plane settings.");
```

Comment:
I am not quite sure all of the things you can do with these are considered management plane operations. Maybe @heaths would know better.
```suggestion
        var admin = new CommandGroup("admin", "Key Vault administration operations");
```